pandas/tests/io/json/test_ujson.py

```diff
@@ -559,6 +560,16 @@ def test_encode_long_conversion(self):
         assert output == json.dumps(long_input)
         assert long_input == ujson.decode(output)
 
+    @pytest.mark.parametrize("bigNum", [sys.maxsize + 1, -(sys.maxsize + 2)])
+    def test_dumps_ints_larger_than_maxsize(self, bigNum):
+        # GH34395
+        bigNum = sys.maxsize + 1
+        encoding = ujson.encode(bigNum)
+
+        assert str(bigNum) == encoding
+        # ujson.loads to be fixed in the future
```

Comment:
is there an issue for this? don't leave in commented code, rather add an xfail test

Reply:
added the test. Also added xfail tests for this in `test_pandas.py`.